WooCommerce/Classes/ViewRelated/Orders/Order Create/OrderFormViewController.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+final class OrderFormViewController: UIViewController {
```

Comment:
```suggestion
final class OrderCreationFormViewController: UIViewController {
```

Reply:
I've preferred to leave `Creation` part out of VC name because we may later reuse it for editing of drafts, what do you think?
Of course we can name it `OrderCreationForm` now and change to generic `Form` later when it fits 😄

Reply:
Sounds good